src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -87,18 +86,22 @@ private void handleNotify(Collection<ItemStack> items, String dropper) {
         }
 
         if (sendMessage) {
+            final long totalPrice = totalStackValue;
             String notifyMessage = StringUtils.replaceEach(
                 config.lootNotifyMessage(),
                 new String[] { "%USERNAME%", "%LOOT%", "%TOTAL_VALUE%", "%SOURCE%" },
                 new String[] { Utils.getPlayerName(client), lootMessage.toString(), QuantityFormatter.quantityToStackSize(totalStackValue), dropper }
             );
-            createMessage(DinkPluginConfig::lootSendImage, NotificationBody.builder()
-                .content(notifyMessage)
-                .embeds(embeds)
-                .extra(new LootNotificationData(serializedItems, dropper))
-                .screenshotFile("lootImage.png")
-                .type(NotificationType.LOOT)
-                .build());
+            createMessage(
+                config -> config.lootSendImage() && totalPrice >= config.lootImageMinValue(),
```

Comment:
Same as the other min image value checker, perform this outside of the createMessage call for improved readability